src/jit/codegencommon.cpp

```diff
@@ -7298,13 +7298,19 @@ void CodeGen::genProfilingEnterCallback(regNumber initReg, bool* pInitRegZeroed)
         return;
     }
 
-#if defined(_TARGET_AMD64_) && !defined(UNIX_AMD64_ABI) // No profiling for System V systems yet.
+#if defined(_TARGET_AMD64_)
     unsigned   varNum;
     LclVarDsc* varDsc;
 
+#if !defined(UNIX_AMD64_ABI)
     // Since the method needs to make a profiler callback, it should have out-going arg space allocated.
     noway_assert(compiler->lvaOutgoingArgSpaceVar != BAD_VAR_NUM);
     noway_assert(compiler->lvaOutgoingArgSpaceSize >= (4 * REGSIZE_BYTES));
+#endif
+
+#if defined(UNIX_AMD64_ABI)
+    regSet.AddMaskVars(REG_R10 | REG_R11 | REG_R12 | REG_R13);
```

Comment:
Since R12 and R13 are callee saved registers, I wonder if this is enough to make the jitter store them.
cc: @CarolEidt